web_patient/src/components/Forms/AddEditActivityForm.tsx

```diff
@@ -1065,33 +1078,34 @@ export const AddEditActivityForm: FunctionComponent<IAddEditActivityFormProps> =
                             )}
                             ampm={true}
                             views={['hours']}
+                            disableCloseOnSelect
```

Comment:
@jayfo The only change in this commit. Rest is all formatting. 

Note that in desktop mode, user needs to click somewhere outside the time picker modal for confirmation. In mobile mode, there is `Cancel` and `Ok` button.